docs/GAS.md

```diff
@@ -45,23 +46,23 @@ multiple ways:
 
 ## Gas overflow potential
 
-CosmWasm gas aims for 1 Teragas/millisecond, i.e. the uint64 range exceeds after
-18 million seconds (5 hours)<sup>1</sup>. Assuming a max supported block
+CosmWasm gas aims for 1 Teragas/second, i.e. the uint64 range exceeds after 18
+million seconds (5000 hours)<sup>1</sup>. Assuming a max supported block
 execution time of 30 seconds, the gas price has to be over-priced by a factor of
-614 (614 Teragas/millisecond) in order to exceed the uint64 range<sup>2</sup>.
+614891 (614891 Teragas/second) in order to exceed the uint64 range<sup>2</sup>.
 Since serious over or underpricing is considered a bug, using uint64 for gas
 measurements is considered safe.
 
-Cosmos SDK gas uses values that are smaller by a factor of 150_000, so those
-don't overflow as well. Since no Cosmos SDK gas values are processed inside of
-this repository, this is not our main concern. However, it's good to know that
-we can safely pass them in uint64 fields, as long as the full range is
-supported. This is the case for the C API as well as
-[JSON numbers](https://www.json.org/) as long as both sides support integers in
-their JSON implementation. Go and Rust do that while many other implementations
-don't support integers, and convert them to IEEE-754 doubles, which has a safe
-integer range up to about 53 bit (e.g. JavaScript and jq).
+Cosmos SDK gas uses values that are smaller by a factor of 150, so those don't
+overflow as well. Since no Cosmos SDK gas values are processed inside of this
+repository, this is not our main concern. However, it's good to know that we can
+safely pass them in uint64 fields, as long as the full range is supported. This
+is the case for the C API as well as [JSON numbers](https://www.json.org/) as
+long as both sides support integers in their JSON implementation. Go and Rust do
+that while many other implementations don't support integers, and convert them
+to IEEE-754 doubles, which has a safe integer range up to about 53 bit (e.g.
+JavaScript and jq).
```

Comment:
Could you add a section briefly explaining the CosmWasm 1 -> 2 change such that we have that available in the future too?